Tests/Marketplace/Tests/marketplace_services_test.py

```diff
@@ -225,3 +225,27 @@ def test_not_existing_changelog_json(self, mocker, dummy_pack):
         mocker.patch("os.path.exists", return_value=False)
         latest_version = dummy_pack.latest_version
         assert latest_version == "1.0.0"
+
+
+class TestChangelogCreation:
+    """ Test class for changelog.json creation step.
+
+    """
+
+    @pytest.fixture(scope="class")
+    def dummy_pack(self):
+        """ dummy pack fixture
+        """
+        dummy_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "test_data")
+        sample_pack = Pack(pack_name="TestPack", pack_path=dummy_path)
+        sample_pack.description = 'Sample description'
+        sample_pack.current_version = '1.0.0'
+        return sample_pack
+
+    def test_prepare_release_notes(self, mocker, dummy_pack):
```

Comment:
we need more tests to Pack.prepare_release_notes. that will cover:
1) when version already exists in changelo
2) Version mismatch detected in release notes and metadata.
3) when change log exist with version 1.0.0  and a new version 1.0.1 is added.(all good)

and add standard test comments with, start, when, expected result